src/request/src/middleware/src/storage/index.js

```diff
@@ -32,7 +32,7 @@ export default class Storage {
       .then(() => MemoryAdapter.load(this.name))
       .then((adapter) => {
         if (isDefined(adapter) === false) {
-          throw new Error('Unable to load a storage adapter.');
+          throw new KinveyError('Unable to load a storage adapter.');
```

Comment:
I agree and this style guide didn't exist when I rewrote the JS SDK with ES6. I will replace all `throws` with `Promise.reject`.